### ISSUE TITLE
Joomla update 3.9.26

### ISSUE THREAD

**Issue by Webdongle** [joomla/joomla-cms#34039]
### Steps to reproduce the issue
J3.9.26 set update channel Testing

### Expected result
A testing version above the version is expected to show i.e. J3.9.27 RC
Site should be updated to J3.9.27 RC

### Actual result
J3.9.26 RC is offered for update
J3.9.26 RC is used to update J3.9.26

### Additional comments
![joomla update 01](https://user-images.githubusercontent.com/1115202/118968464-dddbc300-b963-11eb-98b1-7d2c70183135.jpg)
![joomla update 02](https://user-images.githubusercontent.com/1115202/118968509-e92eee80-b963-11eb-80b9-406022003a9a.jpg)

**Comment by sozzled** [comment]
Ha!  (Apologies for my flippancy.)  A call goes out for people to test J! 3.9.27 RC but fails to provide the automated means to do so!  This should postpone the release of J! 3.9.27 next week and/or remove the "announcement" that appears on the J! forum.

**Comment by PhilETaylor** [comment]
Paging @HLeithner

**Comment by PhilETaylor** [comment]
Maybe this is the problem? No one has updated the XML. 

https://github.com/joomla/update.joomla.org/blob/master/www/core/test/list_test.xml

**Comment by Webdongle** [comment]
Yep that was my guess too but it needs someone with editor privileges to fix it.

**Comment by PhilETaylor** [comment]
Also begs the question why after 20 years we are still not following a checklist approach to releases (or fully automating them)... /facepalm

**Comment by alikon** [comment]
see https://github.com/joomla/update.joomla.org/pull/195

**Comment by Webdongle** [comment]
Will that allow 3.9.26 update to 3.9.27RC on the Test channel?

**Comment by HLeithner** [comment]
> Will that allow 3.9.26 update to 3.9.27RC on the Test channel?

Yes, it's ready.

**Comment by HLeithner** [comment]
> Also begs the question why after 20 years we are still not following a checklist approach to releases (or fully automating them)... /facepalm

No idea why you attack me but 🤷

**Comment by PhilETaylor** [comment]
Why does everyone take things personally. 

Let me rephrase it.. 

Also begs the question why after 20 years [The Joomla Project™] are still not following a checklist approach to releases (or fully automating them)... /facepalm

There is a reason surgeons and pilots use checklists.

**Comment by Webdongle** [comment]
Yes great. 3.9.26 finds 3.9.27rc.  Once updated 3.9.27rc displays 3.10.0-alpha5 on the control panel but the component shows the update url as https://github.com/joomla/joomla-cms/releases/download/3.9.27-rc/Joomla_3.9.27-rc-Release_Candidate-Update_Package.zip
So now I am confused.  Has 3.10.0-alpha5 been started before 3.9.27 stable is released? and why say the update is available when it isn't linked to in the update component?

**Comment by brianteeman** [comment]
yes

**Comment by Webdongle** [comment]
@brianteeman 
Thank you for answering 1/2 the question

**Comment by brianteeman** [comment]
I answered the original question not the one you added afterwards

**Comment by Webdongle** [comment]
The edit was done before you answered so once again thank you for answering 1/2 the question.

Now can somebody explain (before this thread gets bogged down with stalkers) why the message in the control panel says 3.10.0-alpha5 is available but the update component offers 3.9.27rc as an update for 3.9.27rc?

**Comment by brianteeman** [comment]
troll

**Comment by Webdongle** [comment]
Yes you are

**Comment by sozzled** [comment]
> ... can somebody explain (before this thread gets bogged down with stalkers) why the message in the control panel says 3.10.0-alpha5 is available but the update component offers 3.9.27rc as an update for 3.9.27rc?

A bit confusing.  Unfortunately this sounds like a slightly different problem but I don't know anything about it.

My earlier observation remains:

> This should postpone the release of J! 3.9.27 next week and/or remove the "announcement" that appears on the J! forum.
